invenio_sequencegenerator/utils.py

```diff
@@ -1,17 +1,45 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2016-2018 CERN.
+# Copyright (C) 2016-2019 CERN.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
-
 """Utilities for template strings."""
 
 from __future__ import absolute_import, print_function
 
 import re
 
+from flask import current_app
+from werkzeug.utils import import_string
+
+from ._compat import string_types
```

Comment:
I think you could use [six](https://pythonhosted.org/six/#six.string_types)

Reply:
You are definitely right, on the other hand I think it's not worth adding yet another dependency just to get the `string_types` right, I believe we use this same pattern in [some other places](https://github.com/search?q=org%3Ainveniosoftware+org%3Ainveniosoftware+_compat&type=Code) across Invenio.
What do you think?